template-README.md

```diff
@@ -256,7 +279,7 @@ The code is licensed under a MIT/BSD/GPL [choose one!] license. See [LICENSE.txt
 - Download the data files referenced above. Each should be stored in the prepared subdirectories of `data/`, in the format that you download them in. Do not unzip. Scripts are provided in each directory to download the public-use files. Confidential data files requested as part of your FSRDC project will appear in the `/data` folder. No further action is needed on the replicator's part.
 - Run `programs/01_main.do` to run all steps in sequence.
 
-### Details
+### Details on various programs
 
```

Comment:
yeah again shold probably make clear that those are examples?

Reply:
Again, it says "Examples follow". NOt sure how else you want that to be made clear. Open to suggestions, but for now leaving it as-is.

Reply:
ok sorry my bad - i didn't look at the part which was not expanded in the diff because it had not changed. all good.